kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -141,98 +143,11 @@ def test_config_path_move(self):
             do_xcom_push=False,
             config_file=new_config_path,
         )
-        context = self.create_context(k)
+        context = create_context(k)
         k.execute(context)
         actual_pod = self.api_client.sanitize_for_serialization(k.pod)
         self.assertEqual(self.expected_pod, actual_pod)
 
-    @mock.patch("airflow.kubernetes.pod_launcher.PodLauncher.start_pod")
-    @mock.patch("airflow.kubernetes.pod_launcher.PodLauncher.monitor_pod")
-    @mock.patch("airflow.kubernetes.kube_client.get_kube_client")
-    def test_config_path(self, client_mock, monitor_mock, start_mock):  # pylint: disable=unused-argument
```

Comment:
Why are these tests being deleted?

Reply:
I moved all the kuberenetes tests (those that only trigger something in a cluster") to "kubernetes_tests" so they are treated separately and never run inside the regular tests we have. I got rid  of kubernetes runtime - basically the CI image is not used at all for those tests.